src/vma/util/vma_stats.h

```diff
@@ -188,6 +189,10 @@ typedef struct {
 	uint32_t                     n_tx_ready_byte_count;
 	socket_counters_t            counters;
 	std::bitset<MC_TABLE_SIZE>   mc_grp_map;
+	ring_logic_t		     ring_alloc_logic_rx;
+	ring_logic_t		     ring_alloc_logic_tx;
+	uint64_t		     ring_user_id_rx;
+	uint64_t		     ring_user_id_tx;
```

Comment:
Fix indentation, include ring_user_id_rx/ring_user_id_tx in erase() below.

Reply:
Done.